generator/template_test.go

```diff
@@ -91,6 +93,7 @@ const baseTpl = `
 		ID int64 ` + "`pk:\"autoincr\"`" + `
 		Foo string
 		Bar *string
+		Baz int64
```

Comment:
it increase the coverage of the FindBy because now it is considered the sortable properties